sdk/storage/Azure.Storage.Blobs/src/Models/PageBlobRequestConditions.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text;
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Specifies page blob specific access conditions.
+    /// </summary>
+    public class PageBlobRequestConditions : BlobRequestConditions
+    {
+        /// <summary>
+        /// IfSequenceNumberLessThan ensures that the page blob operation
+        /// succeeds only if the blob's sequence number is less than a value.
+        /// </summary>
+        public long? IfSequenceNumberLessThan { get; set; }
+
+        /// <summary>
+        /// IfSequenceNumberLessThanOrEqual ensures that the page blob
+        /// operation succeeds only if the blob's sequence number is less than
+        /// or equal to a value.
+        /// </summary>
+        public long? IfSequenceNumberLessThanOrEqual { get; set; }
+
+        /// <summary>
+        /// IfSequenceNumberEqual ensures that the page blob operation
+        /// succeeds only if the blob's sequence number is equal to a value.
+        /// </summary>
+        public long? IfSequenceNumberEqual { get; set; }
+
+        /// <summary>
+        /// Collect any request conditions.  Conditions should be separated by
+        /// a semicolon.
+        /// </summary>
+        /// <param name="conditions">The collected conditions.</param>
+        internal override void AddConditions(StringBuilder conditions)
```

Comment:
These are used by the base ToString.  Instead of having `[BlobRequestConditions:IfMatch=null;IfNoneMatch=null;...]` we'll just list the conditions that are actually present so it'll be `[BlobRequestConditions:IfMatch=...;LeaseId=...]` if you're only using IfMatch/LeaseId.